tests/test-cases/Group3-Docker-Compose/3-03-Docker-Compose-Basic.robot

```diff
@@ -64,39 +62,41 @@ Compose Up while another container is running (ps filtering related)
     Log  ${out}
     Should Be Equal As Integers  ${rc}  0
 
-    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{VCH-PARAMS} -f basic-compose.yml up -d
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f basic-compose.yml up -d
     Log  ${out}
     Should Be Equal As Integers  ${rc}  0
 
 Compose Up with link
     Run  echo '${link-yml}' > link-compose.yml
-    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{VCH-PARAMS} --file link-compose.yml up -d
+    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --file link-compose.yml up -d
     Log  ${output}
     Should Be Equal As Integers  ${rc}  0
-    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{VCH-PARAMS} --file link-compose.yml logs
+    # Wait for application to come up before checking logs
+    Sleep  40
```

Comment:
I sent this to Cheng, but we should replace this giant sleep with something like:
Create a new keyword:
```
Check Compose Logs
     ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --file link-compose.yml logs
      Should Be Equal As Integers  ${rc}  0
      Should Contain  ${output}  PING aaa
      Should Not Contain  ${output}  bad address 'aaa'
```
Then in the test case:
```Wait Until Keyword Succeeds  10x  10s  Check Compose Logs```